sdk/src/SDK.ts

```diff
@@ -12,6 +12,7 @@ import Platform, {
     SendOptions,
 } from './platform/Platform';
 import {AuthData} from './platform/Auth';
+import {Server} from 'http';
```

Comment:
Do we need this?

Reply:
No It's not required, It was a typo have removed.